Makefile

```diff
@@ -82,7 +82,7 @@ benchmark:
 .PHONY: test-with-cover
 test-with-cover:
 	@echo Verifying that all packages have test files to count in coverage
-	@internal/buildscripts/check-test-files.sh $(subst go.opentelemetry.io/collector/,./,$(ALL_PKGS))
+	@internal/buildscripts/check-test-files.sh $(subst go.opentelemetry.io/collector,.,$(ALL_PKGS))
```

Comment:
fyi this had to change because code was added to the root directory so the substitution wouldn't match properly in that case because of the trailing slash.

Reply:
First answer is always "No" :) Can we avoid that?

Reply:
I don't think there's any downside to changing this.